build_docs.pl

```diff
@@ -390,24 +392,28 @@ sub init_repos {
     my $tracker_path = $Conf->{paths}{branch_tracker}
         or die "Missing <paths.branch_tracker> in config";
 
+    my $reference_dir = dir($Opts->{reference})->absolute;
+    die "Missing reference directory $reference_dir" unless -e $reference_dir;
```

Comment:
Two things:

1. Maybe I am missing some context (or my perl is just too rusty) but this seems like it's a pretty non-optional option if it's going to `die` on this?
2. Looking at the diff I am guessing it was already like that but what on earth happened to this sub's indentation? It is wrong in the PR diff view, but _looks_ right in raw view, which makes me suspect tabs/spaces mixing is going on maybe?

Reply:
RE indentation: Bleh! It looks right locally which is super odd.

RE non-optional: I'll fix it! That'd have broken everything..... I need to test this some more too.